### PR TITLE
Bump cluster-up to include k8s-1.18

### PR DESCRIPTION
**Pull request by dhiller**
**What this PR does / why we need it**:
This PR updates kubevirtci/cluster-up which includes k8s-1.18 provider.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by dhiller** [comment]
See project-infra#447 for test lane addition.

**Comment by jean-edouard** [comment]
@dhiller @oshoval I tried the new 1.17 lane (from the updated images.sh) locally, and it fails with `Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39`.
Looks like that is something that was fixed in kubevirtci (with `export DOCKER_API_VERSION=1.39`) but not in kubevirt.

**Comment by oshoval** [comment]
> @dhiller @oshoval I tried the new 1.17 lane (from the updated images.sh) locally, and it fails with `Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39`.
> Looks like that is something that was fixed in kubevirtci (with `export DOCKER_API_VERSION=1.39`) but not in kubevirt.

Hi, is Kubevirt bumped kubevirtci ?
It might be the reason, because this is one of the things i fixed in kubevirtci

**Comment by jean-edouard** [comment]
In kubevirtci, the fix is in `provision.sh`, so that won't have an impact on kubevirt... Some non-kubevirtci code probably needs fixing.

**Comment by oshoval** [comment]
> In kubevirtci, the fix is in `provision.sh`, so that won't have an impact on kubevirt... Some non-kubevirtci code probably needs fixing.

bump kubevirtci in kubevirt do fix it, i am testing it while we speak
the provision creates a container image, the scripts with the fix are within,
the image hash is updated in Kubevirt as part of the bump,

**Comment by dhiller** [comment]
> @dhiller @oshoval I tried the new 1.17 lane (from the updated images.sh) locally, and it fails with `Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39`.
> Looks like that is something that was fixed in kubevirtci (with `export DOCKER_API_VERSION=1.39`) but not in kubevirt.

Oh, interesting, I wasn't aware that I would have had to bump the new images also? I thought the one who had fixed this in 1.17 had done that?

**Comment by dhiller** [comment]
But we can do this als in this PR if you need this?

**Comment by dhiller** [comment]
> But we can do this also in this PR if you need this?

Sorry, I'm confused. Please disregard everything I said :-/ **Of course we can't do it in this PR**

**Comment by dhiller** [comment]
> @dhiller @oshoval I tried the new 1.17 lane (from the updated images.sh) locally, and it fails with `Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39`.
> Looks like that is something that was fixed in kubevirtci (with `export DOCKER_API_VERSION=1.39`) but not in kubevirt.

@jean-edouard did you really mean the 1.17? I guess not?

**Comment by jean-edouard** [comment]
Yes I do mean 1.17, your PR updates to the one pushed by https://github.com/kubevirt/kubevirtci/pull/366

**Comment by dhiller** [comment]
> Yes I do mean 1.17, your PR updates to the one pushed by [kubevirt/kubevirtci#366](https://github.com/kubevirt/kubevirtci/pull/366)

So at least in CI it seems to be working now: https://prow.apps.ovirt.org/log?id=1265630825072824320&job=pull-kubevirt-e2e-k8s-1.17

**Comment by oshoval** [comment]
> > Yes I do mean 1.17, your PR updates to the one pushed by [kubevirt/kubevirtci#366](https://github.com/kubevirt/kubevirtci/pull/366)
> 
> So at least in CI it seems to be working now: https://prow.apps.ovirt.org/log?id=1265630825072824320&job=pull-kubevirt-e2e-k8s-1.17

i think it doesnt work, the error is printed there in loops
looks like prow upgraded to docker-ce 19.03.09 which had the bug i believe?

see here please
https://github.com/docker/cli/issues/2533#issuecomment-633585349

i would suggest to try one of the workaround that appear on
https://github.com/kubernetes-sigs/kubespray/issues/6160
meanwhile, for prow itself

**Comment by dhiller** [comment]
Looks like we have some problems with the 1.17 provider. Looking at this again in this PR: kubevirt/kubevirtci#369

**Comment by dhiller** [comment]
/hold

there's an error loop when running the [test lane for 1.17](https://prow.apps.ovirt.org/log?id=1265933326129041408&job=pull-kubevirt-e2e-k8s-1.17) that needs to get resolved first.

```
...
2020/05/28 09:22:33 Successfully pushed Docker image to localhost:32775/kubevirt/kv-virt-handler:devel
INFO: Invocation ID: 486d9a8e-0826-42e4-915d-179ee7299dc4
Loading: 
Loading: 0 packages loaded
Analyzing: target //:push-virt-launcher (0 packages loaded, 0 targets configured)
WARNING: Download from https://copr-be.cloud.fedoraproject.org/results/%40virtmaint-sig/for-kubevirt/fedora-30-x86_64/01034621-libvirt/libvirt-devel-5.0.0-2.fc30.x86_64.rpm failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException GET returned 404 Not Found
WARNING: Download from https://copr-be.cloud.fedoraproject.org/results/%40virtmaint-sig/for-kubevirt/fedora-30-x86_64/01034621-libvirt/libvirt-libs-5.0.0-2.fc30.x86_64.rpm failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException GET returned 404 Not Found
INFO: Analyzed target //:push-virt-launcher (0 packages loaded, 117 targets configured).
INFO: Found 1 target...
[0 / 15] [Prepa] Expanding template push-virt-launcher ... (2 actions, 0 running)
Target //:push-virt-launcher up-to-date:
  bazel-bin/push-virt-launcher.digest
  bazel-bin/push-virt-launcher
INFO: Elapsed time: 0.390s, Critical Path: 0.12s
INFO: 0 processes.
INFO: Build completed successfully, 2 total actions
INFO: Running command line: bazel-bin/push-virt-launcher
INFO: Build completed successfully, 2 total actions
2020/05/28 09:22:35 Successfully pushed Docker image to localhost:32775/kubevirt/kv-virt-launcher:devel
go version go1.12.8 linux/amd64

go version go1.12.8 linux/amd64
2020/05/28 09:23:09 Waiting for host: 192.168.66.101:22
2020/05/28 09:23:09 Connected to tcp://192.168.66.101:22
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
Error response from daemon: client version 1.40 is too new. Maximum supported API version is 1.39
...
```

@jean-edouard has [found a fix for this](https://github.com/jean-edouard/kubevirt/commit/1d41270c73c08dd1c072580bb5bc035c814e2ffa), this needs to be integrated properly

**Comment by dhiller** [comment]
So the fix is waiting in this PR: kubevirt/kubevirtci#373 . The quay outage prohibits the jobs succeeding, so we have to wait.

**Comment by dhiller** [comment]
Bumped and rebase yesterday, should be good to go now. Let's see how the k8s-1.17 works out.

**Comment by dhiller** [comment]
I see one flaky test on the 1.17 lane, one on the 1.17 lane and three on the ipv6 lane.

File: .travis.yml [hunk 1]
```diff
@@ -15,7 +15,6 @@ install:
 - git reset --hard
 
 script:
-- set -e
```
Comment (oshoval) [inline .travis.yml:18]: Soon it will be removed :)

Comment (dhiller) [reply .travis.yml]: Removed.

Comment (dhiller) [reply .travis.yml]: Removed.

**Comment by dhiller** [comment]
ipv6 lane flaked: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3475/pull-kubevirt-e2e-kind-k8s-1.17.0-ipv6/1268429205146701824

**Comment by rmohr** [comment]
@dhiller do we have also a 1.18 lane triggered already?

**Comment by dhiller** [comment]
> @dhiller do we have also a 1.18 lane triggered already?

I have created the job already, din't have a look yet honestly. WIll check how status is there.

**Comment by dhiller** [comment]
@rmohr didn't find it directly, as I set it to skip reports, and always_run to false, but looks good for the run I triggered this morning: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/3475/pull-kubevirt-e2e-k8s-1.18/1268429581266718720

**Comment by dhiller** [comment]
@jean-edouard you won :)

**Pull request closed without merging**